addons/sale_timesheet_margin/tests/test_sale_order.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+class TestSaleTimesheetMargin():
+
+    @classmethod
+    def setUpClass(cls):
+        pass
```

Comment:
I believe your test isn't triggered (otherwise it would fail, as it doesn't follow the structure of odoo tests).
Did you at least take a look at other tests to create the base of your test class ?

Reply:
I did a bit of cleaning before committing to be minimalist. But it was wrong...